server.cjs

```diff
@@ -169,7 +172,7 @@
 
     request.get(
       {
-        url: url.format(remoteUrl),
+        url: remoteUrl.toString(),
```

Comment:
@ggetz Since the `url.format` is marked as legacy, I think just using `toString()` is fine here, given how the URL is built previously. Not sure if there are cases where this may fail though.

Reply:
:+1: There's a [non-legacy `format`](https://nodejs.org/api/url.html#urlformaturl-options), but looks like that without any options it will produce the same results as `toString`.